test/authority_test.go

```diff
@@ -33,44 +33,48 @@ import (
 	testpb "google.golang.org/grpc/test/grpc_testing"
 )
 
+func authorityChecker(expectedAuthority string) func(context.Context, *testpb.Empty) (*testpb.Empty, error) {
+	return func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error) {
+		md, ok := metadata.FromIncomingContext(ctx)
+		if !ok {
+			return nil, status.Error(codes.InvalidArgument, "failed to parse metadata")
+		}
+		auths, ok := md[":authority"]
+		if !ok {
+			return nil, status.Error(codes.InvalidArgument, "no authority header")
+		}
+		if len(auths) < 1 {
```

Comment:
`!=` (and update error to print `auths`)

Reply:
Made this change.